README.md

```diff
@@ -24,120 +25,125 @@
 
 1. Checkout the source code.
 
-  ```sh
-  mkdir -p ~/code && cd $_
-  git clone git@github.com:tystakartografen/cogs3.git
-  ```
+   ```sh
+   mkdir -p ~/code && cd $_
+   git clone git@github.com:tystakartografen/cogs3.git
+   ```
 
-2. Create a virtual environment.
+1. Create a virtual environment.
```

Comment:
Item numbers need fixing.

Reply:
Do they render incorrectly? We had a problem early on but github does show them correctly in the current phase-2 branch (in Ed's repo).